tools/autograd/gen_python_functions.py

```diff
@@ -404,6 +404,7 @@ def get_cpp_formal(arg, ensure_temp_safe=True):
     'std::tuple<Tensor,Tensor,Tensor,Tensor,int64_t>',
     'std::tuple<Tensor,Tensor,double,Tensor,int64_t>',
     'std::tuple<double,int64_t>',
+    'std::tuple<double,double>',
```

Comment:
Is this related to your change?

Reply:
yes, this adds a new combination of return types that is not currently supported.